chore: add atomic cli unit & e2e tests, add atomic documentation and prepare for release (#564)
coveo/cli: packages/create-atomic/README.md

npm install -g @coveo/create-atomic
create-atomic myapp
```

> Running the project directly, you will be prompted in order to fill out your organization's authentication information.

Reviewer: ```suggestion
If you run the project directly, you will be prompted to fill out your organization's authentication information.
```
